ReactiveCocoa/Swift/DynamicProperty.swift

```diff
@@ -52,24 +54,12 @@ import enum Result.NoError
 	public init(object: NSObject?, keyPath: String) {
```

Comment:
I don't know either. But how do you think about hiding the initialisers and instead exposing `NSObject.property(forKeyPath:...)`?